Sources/Extensions/SwiftStdlib/StringExtensions.swift

```diff
@@ -231,6 +231,13 @@ public extension String {
 	public var length: Int {
 		return characters.count
 	}
+    /// SwifterSwift: Returns a localized string.
+    ///
+    ///        "Hello!".localized -> Hallo!
+    ///
+    public var localized: String {
+        return NSLocalizedString(self, comment: "")
+    }
```

Comment:
I prefer having a single function here with a default value for comment like below:
```swift
func localized(comment: String = "") {
    ...
}
```

Reply:
Thank you for the suggestion. However, I frequently use localized strings without comments and would prefer "HelloWorld".localized to "HelloWorld".localized().